public/media_engagement/styles.css

```diff
@@ -21,11 +21,14 @@
     background-color: #B6D4E7;
     
 }
-.btn-logout{
+.txt-brown {
+    color: #8d3538; 
+}
+.btn-custom{
```

Comment:
Can you give this a slightly more descriptive name? 

e.g. something like "avf-button" or "btn-brown"